ctapipe/image/muon/fitting.py

```diff
@@ -65,394 +51,6 @@ def kundu_chaudhuri_circle_fit(x, y, weights):
-        distance of the muon impact point from the mirror center
-    pixel_fov: float
-        field of view of the camera pixels in radian
-    mirror_radius: float
-        radius of the telescope mirror (circle approximation)
-    lambda1: float
-        lower integration limit over the cherenkov spectrum in meters
-    lambda2: float
-        upper integration limit over the cherenkov spectrum in meters
-
-    Returns
-    -------
-    light_density: float or array-like
-
-    """
-
-    return (
-        0.5
-        * const.fine_structure
-        * cherenkov_integral(lambda1, lambda2)
-        * pixel_fov
-        / cherenkov_angle
-        * np.sin(2 * cherenkov_angle)
-        * mirror_integration_distance(phi, phi_max, impact_parameter, mirror_radius)
-    )
-
-
-def expected_pixel_light_content(
-    pixel_x,
-    pixel_y,
-    center_x,
-    center_y,
-    phi_max,
-    cherenkov_angle,
-    impact_parameter,
-    sigma_psf,
-    pixel_fov,
-    pixel_diameter,
-    mirror_radius,
-    focal_length,
-    lambda1=300e-9,
-    lambda2=900e-9,
-):
-    """
-    Calculate the expected light content of each pixel for a muon ring with the
-    given properties
-
-    Parameters
-    ----------
-    pixel_x: array-like
-        x coordinates of the pixels
-    pixel_y: array-like
-        y coordinates of the pixels
-    center_x: float
-        x coordinate of the ring center, for example estimated by psf_likelihood_fit
-    center_y: float
-        y coordinate of the ring center, for example estimated by psf_likelihood_fit
-    phi_max: float
-        position of the light maximum
-    cherenkov_angle: float
-        cherenkov_angle of the muon light
-    impact_parameter: float
-        distance of the muon impact point from the mirror center
-    sigma_pdf: float
-        standard deviation for the gaussian profile of the ring
-    pixel_fov: float
-        field of view of the camera pixels in radian
-    pixel_diameter: float
-        diameter of the pixels
-    mirror_radius: float
-        radius of the telescope mirror (circle approximation)
-    focal_length: float
-        focal legth of the telescope
-    lambda1: float
-        lower integration limit over the cherenkov spectrum in meters
-    lambda2: float
-        upper integration limit over the cherenkov spectrum in meters
-
-    Returns
-    -------
-    pe_charge: array-like
-        number of photons for each pixel given in pixel_x, pixel_y
-    """
-    phi = np.arctan2(pixel_y - center_y, pixel_x - center_x)
-    pixel_r = np.sqrt((pixel_x - center_x) ** 2 + (pixel_y - center_y) ** 2)
-    ring_radius = cherenkov_angle * focal_length
-
-    light = radial_light_intensity(
-        phi,
-        phi_max,
-        cherenkov_angle,
-        impact_parameter,
-        pixel_fov,
-        mirror_radius,
-        lambda1,
-        lambda2,
-    )
-
-    result = light * pixel_diameter * norm.pdf(pixel_r, ring_radius, sigma_psf)
-    return result
-
-
-def efficiency_fit(
```

Comment:
isn't this `efficiency_fit` the one used to calculate the `muon_efficiency` given as the output of the muon code?

Reply:
No, this is a different, in tested function doing the same thing.